magento/magento2
# Back port pull #19094

### Description (#19085)
provided fix uses PHP translate instead of not working JS translate

Original PR #19094  

### Fixed Issues (if relevant)
1. #19085:Translation in tier_price.phtml not working

### Manual testing scenarios (*)
1. Configure tier prices for configurable product (for his simples)
2. Change translates for phrases in Magento/ConfigurableProduct/i18n/en_US.csv:
"Buy %1 for %2 each and","Buy %1 for %2 each and TEST"
save,save-TEST
3. Flush caches
4. Go to PDP
5. Choose option of configurable with tier prices
6. Check translates

### Contribution checklist (*)
